db/migrations/20230207110912_create_orders_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE orders (
+  id int PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  price_amount int,
+  delivery_price decimal,
+  delivery_id int,
```

Comment:
어떤 유저의 orders 인지 확인하기 위해 user_id 컬럼을 추가해주면 좋을 것 같습니다.

```suggestion
  id int PRIMARY KEY NOT NULL AUTO_INCREMENT,
  price_amount int,
  delivery_price decimal,
  delivery_id int,
  user_id int
```